socorro/external/rabbitmq/crashstorage.py

```diff
@@ -234,10 +238,10 @@ def _consume_acknowledgement_queue(self):
             while True:
                 crash_id_to_be_acknowledged = \
                     self.acknowledgment_queue.get_nowait()
-                #self.config.logger.debug(
-                    #'RabbitMQCrashStorage set to acknowledge %s',
-                    #crash_id_to_be_acknowledged
-                #)
+                # self.config.logger.debug(
+                #     'RabbitMQCrashStorage set to acknowledge %s',
+                #     crash_id_to_be_acknowledged
+                # )
```

Comment:
Oops. Ignore this change. I couldn't help myself but I later realized the whole file is full of pep8 violations that I don't want to fix now.